### PR TITLE
Product changelog january new

### PR DESCRIPTION
**Pull request by trichards-hiro**
## Description

A new January Release Roundup was published, so adding this to the Docs page.

### CODE REVIEW

File: docs/references/changelog-january.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+- Stacks 2.1 now supports Clarinet 1.4.1, Hiro Explorer, and Stacks.js.
+- Subnets has been improved with an updated Subnets contract compatible with Clarity2.
+- A new trait for NFTs and FTs has been added on the subnet, which specifies a required name and signature for registration. This update ensures the Stacks Blockchain API always knows the correct name for the mint function.
+
+## Additional bitcoin support across Hiro products
+New API endpoints are currently being developed to support Bitcoin-related queries and a Stacks.js client library to connect Bitcoin Core RPC. These improvements enable STX-to-BTC and BTC-to-STX to address conversions.
+
+## Addition*al resources
+For more detailed information on these updates, please refer to the following resources:
```
Comment (LakshmiLavanyaKasturi) [inline docs/references/changelog-january.md:33]: All the resources need to be bulletted list.

File: docs/references/changelog-january.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- an overview of Stacks 2.1 Upgrades.
+- a new Deployment Plans feature guide.
+- additional references for: Hiro Archive and Stacks Connect.
+- Hiro Contributors Guide that describes how to contribute to the technical documentation.
+
+## Empowering developers to do their best work
+Hiro wants to empower developers to do their best work with the latest updates and enhancements to the Hiro product offerings. The Stacks 2.1 upgrade, and its support across different products, include:
+
+- an API release candidate (v7.0.0) that supports Stacks 2.1, including a resolution for a known issue (genesis sync).
```
Comment (LakshmiLavanyaKasturi) [inline docs/references/changelog-january.md:21]: an can be removed may be?

File: docs/references/changelog-january.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+**Note:** This API release candidate is currently in beta.
+
+- Stacks 2.1 now supports Clarinet 1.4.1, Hiro Explorer, and Stacks.js.
+- Subnets has been improved with an updated Subnets contract compatible with Clarity2.
+- A new trait for NFTs and FTs has been added on the subnet, which specifies a required name and signature for registration. This update ensures the Stacks Blockchain API always knows the correct name for the mint function.
+
+## Additional bitcoin support across Hiro products
+New API endpoints are currently being developed to support Bitcoin-related queries and a Stacks.js client library to connect Bitcoin Core RPC. These improvements enable STX-to-BTC and BTC-to-STX to address conversions.
+
+## Addition*al resources
```
Comment (LakshmiLavanyaKasturi) [inline docs/references/changelog-january.md:32]: Typo perhaps. -> Additional

File: docs/references/changelog-january.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- a new Deployment Plans feature guide.
+- additional references for: Hiro Archive and Stacks Connect.
+- Hiro Contributors Guide that describes how to contribute to the technical documentation.
+
+## Empowering developers to do their best work
+Hiro wants to empower developers to do their best work with the latest updates and enhancements to the Hiro product offerings. The Stacks 2.1 upgrade, and its support across different products, include:
+
+- an API release candidate (v7.0.0) that supports Stacks 2.1, including a resolution for a known issue (genesis sync).
+
+**Note:** This API release candidate is currently in beta.
```
Comment (LakshmiLavanyaKasturi) [inline docs/references/changelog-january.md:23]: Note to follow our standards. 

:::note
<content>
:::

File: docs/references/changelog-january.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## Overview
+As Hiro works to provide the most complete and up-to-date information on product releases and updates, the format of the Product Changelog has been altered slightly to make the content easier to read and understand. Instead of deep dives into each product, the focus has shifted to the developer experience and how these updates and changes improve the overall "developer journey."
+
+The sections below describe the changes and updates made in January 2023.
+
+## Developer onboarding
+The Hiro documentation team has been making significant progress updating the technical documentation. The following updates/changes were added:
+
+- an overview of Stacks 2.1 Upgrades.
```
Comment (LakshmiLavanyaKasturi) [inline docs/references/changelog-january.md:13]: an can be removed?

File: docs/references/changelog-january.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- a new Deployment Plans feature guide.
```
Comment (LakshmiLavanyaKasturi) [inline docs/references/changelog-january.md:14]: a can be removed and N in new can be capital

File: docs/references/changelog-january.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- additional references for: Hiro Archive and Stacks Connect.
```
Comment (LakshmiLavanyaKasturi) [inline docs/references/changelog-january.md:15]: additional -> Additional?

File: docs/references/changelog-january.md (hunk 1 above)
Comment (trichards-hiro) [reply docs/references/changelog-january.md:35]: fixed

File: docs/references/changelog-january.md (hunk 2 above)
Comment (trichards-hiro) [reply docs/references/changelog-january.md]: fixed

File: docs/references/changelog-january.md (hunk 3 above)
Comment (trichards-hiro) [reply docs/references/changelog-january.md]: fixed

File: docs/references/changelog-january.md (hunk 4 above)
Comment (trichards-hiro) [reply docs/references/changelog-january.md]: fixed

File: docs/references/changelog-january.md (hunk 5 above)
Comment (trichards-hiro) [reply docs/references/changelog-january.md]: fixed

File: docs/references/changelog-january.md (hunk 7 above)
Comment (trichards-hiro) [reply docs/references/changelog-january.md]: fixed

**Comment by LakshmiLavanyaKasturi** [comment]
LGTM! Please sign the CLA and ship it.

**Pull request merged**